### PR TITLE
14244-Change-Mhr-Draft-to-display-DocumentRegistrationNumber

### PR DESCRIPTION
**Pull request by ChasBoucher**
*Issue #:* /bcgov/entity14244
*Description of changes:*

- Add RegistrationNumber to registration-interface
- Change DocumentId to DocumentRegistration on Child row 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/interfaces/ppr-api-interfaces/registration-interfaces.ts [hunk 1]
```diff
@@ -101,6 +101,7 @@ export interface MhRegistrationSummaryIF {
   changes?: MhRegistrationSummaryIF []
   hasDraft?: boolean
   documentId?: string
+  documentRegistrationNumber?:string
```
Comment (owillborn) [inline ppr-ui/src/interfaces/ppr-api-interfaces/registration-interfaces.ts]: Missing a space?

Comment (dimak1) [inline ppr-ui/src/interfaces/ppr-api-interfaces/registration-interfaces.ts]: We are just adding new field on top of `documentId`? Both will be used?

**Review by dimak1 — APPROVED** [review]

File: ppr-ui/src/interfaces/ppr-api-interfaces/registration-interfaces.ts (hunk 1 above)
Comment (ChasBoucher) [reply ppr-ui/src/interfaces/ppr-api-interfaces/registration-interfaces.ts]: They are separate fields(api returns both)... Both are used in various reports and displays and code...  In this case the MHR table child rows.  Honestly,  I am not quite sure of all the business rules around these two ids(numbers).  I know I was displaying the wrong number and it was not included in any of the interfaces...

**Review by doug-lovett — APPROVED** [review]
Looks good.

**Review by cameron-eyds — APPROVED** [review]

File: ppr-ui/src/interfaces/ppr-api-interfaces/registration-interfaces.ts (hunk 1 above)
Comment (dimak1) [reply ppr-ui/src/interfaces/ppr-api-interfaces/registration-interfaces.ts]: Ok thanks for the clarification.

**Pull request merged**